src/backend/distributed/shared_library_init.c

```diff
@@ -2283,6 +2298,26 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
+		"citus.stats_tenants_limit",
+		gettext_noop("monitor limit"),
+		NULL,
+		&CitusStatsTenantsLimit,
+		10, 1, 100,
```

Comment:
these values seem pretty low, which does not make me super confident that we will actually be able to capture the top tenants

Reply:
Let's set the default to 100, and the maximum to 100000.